infrastructure/src/main/java/mops/controllers/StudentController.java

```diff
@@ -54,10 +57,15 @@ public String uebersicht(KeycloakAuthenticationToken token, Model model, String
   public String fragebogen(KeycloakAuthenticationToken token,
                            Model model, String search, Long veranstaltungId) {
     Veranstaltung veranstaltung = veranstaltungen.getVeranstaltungById(veranstaltungId);
+    Student student = new Student(((KeycloakPrincipal) token.getPrincipal()).getName());
+    List<Fragebogen> notSubmittedFrageboegen = singleSubmitService
+        .notSubmittedFrageboegen(veranstaltung.getFrageboegen(), student);
```

Comment:
Better to just get this directly from the Repository honestly. The Repository can use the Service. We want to keep Logic out of the Controller.